distro/jbossas7/modules/src/main/modules/org/camunda/bpm/camunda-engine/main/module.xml

```diff
@@ -32,6 +30,8 @@
     <module name="org.camunda.commons.camunda-commons-logging" />
     <module name="org.camunda.commons.camunda-commons-utils" />
     <module name="org.camunda.commons.camunda-commons-typed-values" export="true" />
+    <module name="org.camunda.connect.camunda-connect-core" />
+    <module name="org.camunda.connect.camunda-connect-http-client" services="import" />
```

Comment:
Is this import still needed when the connectors are initialized in the subsystem?